www/src/data/sidebars/doc-links.yaml

```diff
@@ -93,26 +93,32 @@
               breadcrumbTitle: ESLint
             - title: Proxying API Requests
               link: /docs/api-proxy/
-        - title: Images, Files, and Video
+        - title: Images, Files & Video
           link: /docs/images-and-files/
           items:
-            - title: Importing Assets Directly Into Files
+            - title: Importing Assets Directly into Files
               link: /docs/importing-assets-into-files/
-              breadcrumbTitle: Importing Assets Into Files
+              breadcrumbTitle: Importing Assets into Files
             - title: Using the Static Folder
               link: /docs/static-folder/
+              breadcrumbTitle: Static Folder
             - title: Using gatsby-image
               link: /docs/using-gatsby-image/
+              breadcrumbTitle: gatsby-image
             - title: Working with Images in Gatsby
               link: /docs/working-with-images/
-            - title: Preoptimizing your Images
+              breadcrumbTitle: Images in Gatsby
+            - title: Preoptimizing Your Images
               link: /docs/preoptimizing-images/
-            - title: Working With Video
+              breadcrumbTitle: Preoptimizing Images
+            - title: Working with Video
               link: /docs/working-with-video/
+              breadcrumbTitle: Video
```

Comment:
```suggestion
              breadcrumbTitle: Video's in Gatsby
```

Reply:
Shouldn't the breadcrumb titles just be shorter names for the actual article title? If "Video" is not suitable as a breadcrumb title then I think we should just leave it to inherit the default title.

Reply:
Disregard the previous comment! I just realized the "in Gatsby" suggestion was to match the previous item's breadcrumb title (Images in Gatsby). I'll revise both the "Videos" and "GIFs" breadcrumb in a new commit shortly.